src/schema/nodes/manuscript.ts

```diff
@@ -28,6 +28,14 @@ export interface ManuscriptAttrs {
   articleType: string
   prototype: string
   primaryLanguageCode: string
+  dates: {
```

Comment:
I don't think we need the `dates` wrapper, we can just have the dates as a flat list under attrs.

Reply:
updated